pkg/commands/git_commands/archive.go

```diff
@@ -0,0 +1,43 @@
+package git_commands
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/utils"
+)
+
+type ArchiveCommands struct {
+	*GitCommon
+}
+
+func NewArchiveCommands(gitCommon *GitCommon) *ArchiveCommands {
+	return &ArchiveCommands{
+		GitCommon: gitCommon,
+	}
+}
+
+func (self *ArchiveCommands) Archive(refName string, archiveName string, prefix string) error {
+	cmdArgs := NewGitCmd("archive").ArgIf(prefix != "", "--prefix", prefix).Arg("-o", archiveName, refName)
```

Comment:
Most of the time we format commands so that each argument is on a separate line, so I'd format this something like:
```
	cmdArgs := NewGitCmd("archive").
		ArgIf(prefix != "", "--prefix", prefix).
		Arg("-o", archiveName).
		Arg(refName)
```